Proofs

dmccoomes opened the pull request:
Updated manuscript with comments and changes from David and Jessica for PDL submission.

jlgodwin committed:
Provide suggested edits for Intro

suggest merge of first two sections

jlgodwin committed:
Suggested edits for Definition of a walk bout

JLG Comments and suggestions for section 2

jlgodwin committed:
Comments and suggestion for section 3

laurenwilner commented on paper.md:
```diff
@@ -52,14 +52,22 @@ authors:
 
 # Summary 
-Walking is the most common form of physical activity and a behavior of key interest for urban planners, health promotion researchers, and rehabilitation medicine practitioners. Personal monitoring with accelerometer and Global Positioning System (GPS) devices is the gold standard measurement approach for walking. However, processing accelerometer and GPS device traces to identify walking is a com ...
+<!-- JLG: Suggest collapsing these two sections and calling it something like Introduction or, more exciting: Walking and Measurement in Public Health Research -->
```
i think this sounds good, i like the new title for it and am fine with editing it as you propose.

laurenwilner commented on paper.md:
```diff
@@ -52,14 +52,22 @@ authors:
+
 A walk bout is defined as a period of activity in which an accelerometer trace indicates movement consistent with walking and GPS traces from the corresponding time period indicate movement through space consistent with walking (e.g., based on speed) as well. 
 
 <!-- DMC: This sentence above is not real clear to me. Isn't the accelerometer (not just the GPS trace) based on speed? I thought the GPS trace was used to define the position or the overall distance travelled to differentiate between walking around the house vs going for a walk? 
```
how's this:

A walk bout is defined as a period of activity in which (1) an accelerometer trace indicates movement consistent with walking (e.g., based on speed) and (2) a GPS traces from the corresponding time period indicate movement through space (e.g., based on distance traveled).

laurenwilner commented on paper.md:
```diff
@@ -78,11 +86,26 @@ By offering a comprehensive set of functions, `walkboutr` empowers researchers a
 
```
i can't comment directly on the paragraph above, but re the comment in line 83, i think that paragraph is good if there is space. If not, feel free to cut it. but i do think its an important summary of the gap this fills.

laurenwilner commented on paper.md:
```diff
@@ -78,11 +86,26 @@ By offering a comprehensive set of functions, `walkboutr` empowers researchers a
 <!-- DMC: would it be useful to report what accelerometer and GPS measurements are capturing?  -->
 
 
+<!-- JLG: Maybe not necessary, but general note: Can we change references to "we" to the package unless it's referring to the researchers themselves. -->
```
yes!

laurenwilner commented on paper.md:
```diff
@@ -110,16 +133,15 @@ We can then classify physical activity bouts as walking or not walking based on
 * has a median speed consistent with that of walking (specific thresholds configurable and outlined below)
 * has sufficient GPS coverage 
 
+<!-- JLG: Title section 3 something like Using the `walkboutr` package -->
```
i like this!

laurenwilner commented on paper.md:
```diff
@@ -110,16 +133,15 @@ We can then classify physical activity bouts as walking or not walking based on
-
-
-
+<!-- JLG: Title section 3.1 something like Identifying physical activity bouts-->
```
i like this!

laurenwilner commented on paper.md:
```diff
@@ -110,16 +133,15 @@ We can then classify physical activity bouts as walking or not walking based on
 First, from accelerometry data alone, we identify physical activity bouts. These bouts indicate periods of time in which the wearer appears to be physically active, though not necessarily walking (e.g., they could be playing a sport or working out). In order to identify physical activity bouts in accordance with the definition given above, `walkboutr` uses a run-length encoding algorithm to ident ...
 
-<!-- The last sentence in the paragraph above is a little confusing to me. If there are 4 or more consecutive inactive epochs, aren't all of them by definition not a bout, not just the final one?  -->
+<!-- DMC: The last sentence in the paragraph above is a little confusing to me. If there are 4 or more consecutive inactive epochs, aren't all of them by definition not a bout, not just the final one?  -->
```
no. this means that the individual's bout that had already begun would have three periods of inactivity at the end, but not 4. if there was not a bout going on, and just several epochs of inactivity, then yes all of them would be not part of a bout. but here, we are saying the first three would be part of the pre-existing bout.

laurenwilner commented on paper.md:
```diff
@@ -140,6 +162,7 @@ Finally, `walkboutr` determines if the user wore their accelerometer for a suffi
 
 After identifying all physical activity bouts and flagging for non-wearing time, the accelerometry data is ready to be merged with GPS data. 
 
+<!--JLG title section 3.2 Merging GPS and accelerometer data-->
```
looks good!

laurenwilner commented on paper.md:
```diff
@@ -150,6 +173,7 @@ Next, `walkboutr` calculates a circle to approximate the distance covered by thi
 
 GPS data are then evaluated for completeness based on whether they have a sufficient number of GPS records. This is assessed both in terms of the number of GPS observations within a bout as well as the ratio of observations that have GPS data. By default, a bout has sufficient GPS coverage if it has at least five GPS observations and at least 20% of the epochs have a paired GPS observation. 
 
+<!-- JLG: Title section 3.3 Identifying walk bouts-->
```
Looks good!

laurenwilner commented on paper.md:
```diff
@@ -164,6 +188,7 @@ The final step in `walkboutr` synthesizes the above information and labels each
 5.	Among remaining physical activity bouts, bouts whose GPS data do not exceed a bounding radius of 66 feet are labeled as dwell bouts (labeled as **dwell_bout**).
 6.	Any remaining physical activity bouts are labeled as walk bouts (labeled as **walk_bout**).
 
+<!-- JLG: need a snippier title for section 3.4, maybe we call it something like Producing walk bout datasets for analysis-->
```
Fair point :) That sounds good!

laurenwilner reviewed (approved):
Hi @dmccoomes @jlgodwin -- this looks good to me! I think i addressed everything. I will reply to your email but I am good with moving forward with all of these changes.

jlgodwin committed:
Change section titles and easy to change things

Input all agreed upon edits, deleted superfluous text that was replaced by agreed upon edits, deleted davids cuts at the end.

What's left is fine-tooth comb over middle section

jlgodwin committed:
Modify section I. Identifying physical activity bouts

jlgodwin committed:
Fix up section II

jlgodwin committed:
Update Fig 1 and add comment about bout end pont

jlgodwin committed:
Fix up section III

Note there are two important questions still in comments in numbered list about defaults for 3. and 4.

jlgodwin committed:
Fix up last two sections

jlgodwin committed:
Fix table references

jlgodwin committed:
Update defaults and address Steve suggested edits

remove "(e.g. a walkability analysis)" from 2nd paragraph of Producing walk bout datasets for analywsis
